src/Services.java

```diff
@@ -0,0 +1,236 @@
+        {
+            System.out.println();
+            System.out.print("User Name: ");
+            userName =  scanner.nextLine().toUpperCase();
+        }
+
+        System.out.print("IBAN: ");
+        IBAN = scanner.nextLine().toUpperCase();
+
+        while(!check.isValidIBAN(IBAN))
+        {
+            System.out.println();
+            System.out.print("IBAN: ");
+            IBAN = scanner.nextLine().toUpperCase();
+        }
+
+        System.out.print("PIN: ");
+        PIN = scanner.nextLine();
+
+        while(!check.isValidPin(PIN))
+        {
+            System.out.println();
+            System.out.print("PIN: ");
+            PIN = scanner.nextLine();
+        }
+
+        Files.write(Paths.get(outfile), (userName+"\t"+ IBAN+"\t"+ PIN).concat("\n").getBytes(), StandardOpenOption.APPEND);
+
+        System.out.println();
+        System.out.println(ANSI_GREEN + "Your account has been created." + ANSI_RESET);
+        System.out.println();
+        return userName;
+    }
+
+    public double getCurrentBalance(String userName) throws IOException {
+        double currentBalance = 0;
+        File file = new File("TransactionHistory.txt");
+        FileReader f = new FileReader(file);
+        BufferedReader b = new BufferedReader(f);
+
+        String line;
+        while((line = b.readLine()) != null) {
+            String[] columns = line.split("\t");
+
+            if(columns[0].equals(userName.toUpperCase()) )
+            {
+                switch (columns[2])
+                {
+                    case "WITHDRAW" : {
+                        currentBalance -= Double.parseDouble(columns[4]);
+                        break;
+                    }
+
+                    case "DEPOSIT" : {
+                        currentBalance += Double.parseDouble(columns[4]);
+                        break;
+                    }
+
+                    case "BILLING" : {
+                        currentBalance -= Double.parseDouble(columns[4]);
+                        break;
+                    }
+                }
+            }
+
+        }
+        f.close();
+
+        return currentBalance;
+    }
+
+    public boolean depositCash(String userName, String cash) throws IOException {
+        if(!check.isValidLongInput(cash))
+        {
+            return false;
+        }
+        else {
+            System.out.println();
+            System.out.println("You have deposited " + cash + " " + currency + " in the current account.");
+
+            destination = "debit account";
+            service = "deposit";
+            saveToFile.saveHistory(userName,date.now().toString(), service, destination, cash, currency);
+            return true;
+        }
+    }
+
+    public boolean withdrawCash(String userName,String cash) throws IOException {
+        if(!check.isValidLongInput(cash))
+        {
+            return false;
+        }
+        else {
+            if(Long.parseLong(cash) > getCurrentBalance(userName))
```

Comment:
if userName is not case sensitive, try to use toLower/Upper, just to be sure you will not have problems